hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -165,10 +165,7 @@ protected void doWrite(HoodieRecord record, Schema schema, TypedProperties props
       // record successful.
       record.deflate();
     } catch (Throwable t) {
-      // Not throwing exception from here, since we don't want to fail the entire job
-      // for a single record
-      writeStatus.markFailure(record, t, recordMetadata);
```

Comment:
Does any other code pieces requires this stats?(`markFailure`) I'm also confused why we only need to do a mark here

Reply:
after the process， job would stop，seem not need the stats any more @boneanxs

Reply:
add config to judge whether error out when write failed @boneanxs @danny0405

Reply:
1. May also need to add all other Handles that MergeHandle and check other usages using `markFailure`
2. Make `config.ignoreErrorWhenWriteFailed()` to be an internal variable of `HoodieWriteHandle` is more suitable

Reply:
ok，thanks for your advices @boneanxs

Reply:
addressed，add all Handles using markFailure and add an internal variable of HoodieWriteHandle @boneanxs